pkgs/by-name/li/libkrun-efi/package.nix

```diff
@@ -0,0 +1,86 @@
+{
+  cargo,
+  fetchFromGitHub,
+  fetchurl,
+  lib,
+  libepoxy,
+  moltenvk,
+  pkg-config,
+  rustc,
+  rustPlatform,
+  rutabaga_gfx,
+  stdenv,
+  virglrenderer,
+  vulkan-headers,
+  withGpu ? false,
```

Comment:
consider adding `passthru.tests.withGpu = libkrun-efi.override { withGpu = true; }` to define a test case for the alternative configuration.